package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@bedrock/multikey-context",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "Bedrock loader for multi key context",
+  "main": "./lib/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/bedrock-multikey-context.git"
+  },
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "license": "SEE LICENSE IN LICENSE.md",
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-multikey-context/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-multikey-context#readme",
+  "dependencies": {
+    "@digitalbazaar/multikey-context": "^1.0.0"
+  },
+  "peerDependencies": {
+    "@bedrock/core": "^6.0.0",
```

Comment:
@dlongley @gannan08 should I drop this peer dep of `@bedrock/core` or should I keep it?

Reply:
Removed `@bedrock/core` peerDep here: https://github.com/digitalbazaar/bedrock-multikey-context/pull/1/commits/885fc2561defe9267b9586df7fb989080b8acd6a